notebooks/229-distilbert-sequence-classification/229-distilbert-sequence-classification.ipynb

```diff
@@ -0,0 +1,319 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e2b748f3",
+   "metadata": {},
+   "source": [
+    "# Sentiment analysis\n",
+    "\n",
+    "**Sentiment analysis** or **Sequence Classification** is the use of natural language processing, text analysis, computational linguistics, and biometrics to systematically identify, extract, quantify, and study affective states and subjective information. This notebook demonstrates how to convert and run sequence classification model using OpenVINO. "
```

Comment:
@arunimaaaa I'm confused by this. Are you saying Sentiment analysis is using sequence classification? Or that Sentiment Analysis and Sequence Classification are the same thing? Can you point to a document or research paper that explains this?

Reply:
Hi @ryanloney you can check [this](https://huggingface.co/docs/transformers/tasks/sequence_classification) out. 
Basically, on the first version of this notebook, I was [serializing](https://huggingface.co/docs/transformers/serialization) the model using the transformers.onnx function where I was setting the feature parameter as sequence-classification.

Reply:
sentiment analysis is particular case of sequence classification, it classifies sequence of tokens (or sentence) for identifying its emotional color. sequence classification capabilities not limited with this case only (e.g. popular benchmark for NLP - GLUE also uses this task for paraphrase analysis and search for text entailment, verification grammatical structure correctness and linguistic acceptability. It can be considered as particular case of text categorization by topics) Probably we need more clear explanation here, because these 2 tasks are not equal

Reply:
I can just remove the term Sequence Classification and keep Sentiment Analysis

Reply:
I'll just edit the description